purchases/src/main/kotlin/com/revenuecat/purchases/BillingFactory.kt

```diff
@@ -28,7 +29,7 @@ object BillingFactory {
             try {
                 Class.forName("com.revenuecat.purchases.amazon.AmazonBilling")
                     .getConstructor(Context::class.java, Backend::class.java, DeviceCache::class.java)
```

Comment:
The constructor has changed, so this constructor doesn't exist anymore and this line needs to be updated. It is not ideal this compiles but we are instantiating using reflection 😢